changelog.d/20250428_161054_PL-133648-fix-clamav-on-devhost_scriv.md

```diff
@@ -0,0 +1,21 @@
+<!--
+
+A new changelog entry.
+
+Delete placeholder items that do not apply. Empty sections will be removed
+automatically during release.
+
+Leave the XX.XX as is: this is a placeholder and will be automatically filled
+correctly during the release and helps when backporting over multiple platform
+branches.
+
+-->
+
+### Impact
+
+- A bullet item for the Impact category.
```

Comment:
Still not certain whether this needs to be removed by the PR author. We'll see during release.

Reply:
Yeah, currently the author should remove this, but I'd like to change the release scripts because this is a nag that even I run into ... In any case: we catch this during release note editing ...